### ISSUE TITLE
Unable to view previous quiz attempts

### ISSUE THREAD

**Issue by ziobron** [themeum/tutor#8]
How to reproduce:
- Enable ability to view previous quiz attempts in options
- Take z quiz attempt
- View your attempt (it is ok now)
- Go to another lesson or any other link
- Come back again to the quiz page
- Try to view attempt (displays 0 only as in attached screenshot)

![Zrzut ekranu 2020-05-13 o 12 09 23](https://user-images.githubusercontent.com/4595135/81800109-9cc00000-9512-11ea-930a-3400cdd7376c.png)

Interesting thing is that after another attempt I am able to see my previous attempts as long as I stay in this Quiz lesson. Changing lesson causes all attempts to be unavailable.

**Comment by girishboke** [comment]
@ziobron Yes, i just had same issue, for some reason after hard reloading (ctrl + R) twice, able to see the Quiz Attempt.

**Comment by girishboke** [comment]
Hey hi @ziobron,
After struggling for a week, i have found the solution for above mentioned issue.

To solve it, please follow following steps.
1. Go to plugins/tutor-pro/classes
2. update the code in _public function add_attempts_tbody_col($attempt)_.

echo '<a href="'. esc_url( add_query_arg(array('view_quiz_attempt_id' => $attempt->attempt_id), get_permalink() ) ).'">'.__('View Attempt', 'tutor-pro').'</a>';

i hope this will solve your issue as well.
